### PR TITLE
Postman request

### PR DESCRIPTION
**Pull request by BEAdi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44189

## Description
Request data will have the python argument name and not the raw data argument name.

### CODE REVIEW

**Comment by BEAdi** [comment]
failing test before fixing: https://app.circleci.com/pipelines/github/demisto/demisto-sdk/12833/workflows/76445879-86ca-4a44-af82-19cca5af0fc5/jobs/68709

File: demisto_sdk/commands/postman_codegen/postman_codegen.py [hunk 1]
```diff
@@ -89,16 +89,21 @@ def create_body_format(body: Union[dict, list]):
...
+                o[k] = format_value(v, o[k], path + (k,))
             elif isinstance(v, list) and len(v) > 0 and isinstance(v[0], dict):
                 o[k] = [{}]
-                o[k][0] = format_value(v[0], o[k][0])
+                o[k][0] = format_value(v[0], o[k][0], path + (k,))
             else:
                 o[k] = '{' + k + '}'
+                if args:
+                    for arg in args:
+                        if arg.in_object == list(path) and k in arg.name:
```
Comment (PA-Rweins) [inline demisto_sdk/commands/postman_codegen/postman_codegen.py:104]: Why `k in arg.name` and not just check the suffix?

**Review by PA-Rweins — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py [hunk 2]
```diff
@@ -74,6 +75,68 @@ def test_create_body_format_list_of_dicts(self):
             }
         ]
 
+    def test_create_body_format_different_arg_name(self):
+        request_body = {
```
[the remark this answers is not in the document]
Comment (BEAdi) [reply demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py:89]: Done

[the remark this answers is not in the document]
Comment (BEAdi) [reply demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py:89]: Done

File: demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py [hunk 3]
```diff
@@ -74,6 +75,68 @@ def test_create_body_format_list_of_dicts(self):
...
+            "key1": "val1",
```
[the remark this answers is not in the document]
Comment (BEAdi) [reply demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py]: Done

**Review by PA-Rweins — APPROVED** [review]
LGTM

**Review by PA-Rweins — APPROVED** [review]
LGTM

**Pull request merged**